include/openPMD/RecordComponent.hpp

```diff
@@ -108,6 +108,17 @@ class RecordComponent : public BaseRecordComponent
     template< typename T >
     RecordComponent& makeConstant(T);
 
+    /**
+     * Create a dataset with zero extent in each dimension.
+     * Implemented by creating a constant record component with
```

Comment:
potentially add an empty comment line to separate title and brief description in doxygen :)